src/explore-education-statistics-admin/src/components/editable/EditableContentForm.tsx

```diff
@@ -152,6 +163,20 @@ const EditableContentForm = ({
 
             return (
               <Form id={`${id}-form`}>
+                {showAltTextMessage && (
+                  <WarningMessage>
```

Comment:
Given my other comment on adding outlines in content block previews, it might be a good idea to show this warning in the preview mode as well.

We could probably implement this via a DOM selector when the preview renders to trigger the message. The selector would just look for any images with no alt text of course.

Reply:
Yep, showing the message and the red border now.